test/streams/cleanupStream.js

```diff
@@ -74,14 +74,73 @@ tape( 'cleanupStream trims leading 0\'s from house numbers', function(test) {
 
 tape ( 'cleanupStream trims white space in street field', function(test){
   var input = {
-      STREET: '34  West\t 93rd \nst'
+      STREET: '34  West\t 93rd \nSt'
```

Comment:
I guess this is just the test to verify that there is a chance that nothing is changed. All good.